source/manual/rummager-traffic-replay.html.md

```diff
@@ -34,21 +34,25 @@ These indexes are populated by whitehall calling an HTTP API in Rummager.
 We have also setup [Gor][gor] logging for `POST` and `GET` requests so that we
 can replay the traffic.
 
-The logs are stored on the rummager servers (1 file per server) location at:
+The logs are stored on the rummager servers. You will need to run the replay on
+each server.
+
+The location of the logs is:
 
 ```
 /var/log/gor_dump
 ```
 
-A copy of the file should be taken for the restore, as the restore requests
-will be logged to the file. The following command can be used to run the
-restore:
+You must copy the file for the restore, as the restore requests
+will be logged to the file.
+
+The following command can be used to run the restore:
 
 ```bash
-$ sudo goreplay -input-file "20171031.log|6000%" -stats -output-http-stats -output-http "http://localhost:3009/|6000%" -verbose
+$ sudo goreplay -input-file "20171031.log|1000%" -stats -output-http-stats -output-http "http://localhost:3009/|6000%" -verbose
 ```
 
-This runs the restore at 60x the speed it was saved so each hour of logs takes
+This runs the restore at 10x the speed it was saved so each hour of logs takes
```

Comment:
The calculation here is no longer correct (each hour will take 6 minutes to process).

Reply:
👍 thanks Tim. Updated.